templates/components/itilobject/fields_panel.html.twig

```diff
@@ -223,7 +223,7 @@
                {{ fields.dropdownField(
                   'Contract',
                   '_contracts_id',
-                  item.fields['_contracts_id'],
+                  params['_contracts_id']|default(0),
```

Comment:
Why this one would be retrieved from `params` while other fields are retrieved from `item.fields` ?
Maybe there is a bad filtering/assignation logic somewhere else.

Reply:
The field is not present in the $fields class property for a new item so its value is not restored into that array after a page refresh. It is only restored into the $input class property which makes it available through the params twig variable.

Since this field only exists for new tickets I don't know if it makes sense to try and change the logic used to restore saved inputs.